files/image_config/resolv-config/resolv-config.sh

```diff
@@ -14,8 +14,19 @@ start()
 {
     update_symlink
 
+    has_static_mgmt_ip=false
+    redis-dump -d 4 -k "MGMT_INTERFACE|eth0|*" -y > /tmp/dns-mgmt-intf.json
```

Comment:
Can we use redis-cli -n 4 keys *MGMT_INTERFACE|eth0* to verify if mgmt interface is configured. It would be single step and we don't need to use a temporary file